packages/@divvi/mobile/src/storage/keychain.tsx

```diff
@@ -91,3 +91,15 @@ export async function listStoredItems() {
     throw error
   }
 }
+
+export async function clearStoredItems() {
+  const items = await listStoredItems()
```

Comment:
worth sending an analytics event if there are items here so we can see how often this happens?